pkg/tid/cli/command/timesheet/list.go

```diff
@@ -47,7 +49,12 @@ func ListCommand(factory util.Factory) *console.Command {
 		now := timeutil.Date(time.Now())
 
 		if !hasStart {
-			start = timeutil.LastWeekday(time.Monday)
+			weekday, err := timeutil.StringToWeekday(config.Display.FirstWeekDay)
+			if err != nil {
+				fmt.Println(err)
```

Comment:
I notice that the command here deals with errors in one of two ways:

* they are returned and handled elsewhere, or
* a message is printed on `output` and then the execute function stops

I'm not entirely sure which is the right one here, perhaps @SeerUK can weigh in on this one